### PR TITLE
Revert "Remove handheld crew monitor (#24761)"

### PR DESCRIPTION
**Pull request by metalgearsloth**
This reverts commit bbc6d29.

As mentioned on discord with (mirror and emisse?) just make it CMO only and not roundstart for everyone.

https://discord.com/channels/310555209753690112/1202667826427596831/1205804070825365514

:cl:
- add: Added handheld crew monitor back just for CMO.

### CODE REVIEW

File: Resources/Prototypes/Catalog/thief_toolbox_sets.yml
Comment (crazybrain23) [inline Resources/Prototypes/Catalog/thief_toolbox_sets.yml:1]: You added a lot of trailing whitespace to this file.

**Comment by FairlySadPanda** [comment]
If this is CMO loot then should it be a Syndie/Thief objective?

**Comment by Kadeo64** [comment]
> If this is CMO loot then should it be a Syndie/Thief objective?

the syndicate already has crew monitoring technology

**Comment by PJB3005** [comment]
I thought it already *was* CMO only? It was standard protocol for the paramedic to ask the CMO for it at round start.

**Comment by K-Dynamic** [comment]
> I thought it already _was_ CMO only? It was standard protocol for the paramedic to ask the CMO for it at round start.

Probably had to do with the fact they were mapped in some places (Fland had like 3 in total) and could be researched by sci

**Comment by keronshb** [comment]
Not sure if this a good idea. We'd need strict standards to make sure this doesn't get mapped or reused elsewhere - like in a PDA program for example.

**Comment by Hanzdegloker** [comment]
> Not sure if this a good idea. We'd need strict standards to make sure this doesn't get mapped or reused elsewhere - like in a PDA program for example. Or borgs/AI.

Add (do not map) like there exists already on items like the CMO's rubber stamp, simple as.

**Comment by deltanedas** [comment]
suffix: CMO only, do not map

**Comment by Kadeo64** [comment]
cat ears [DO NOT MAP]

File: Resources/Prototypes/Catalog/thief_toolbox_sets.yml [hunk 1]
```diff
@@ -83,21 +83,22 @@
...
   name: thief-backpack-category-communicator-name
   description: thief-backpack-category-communicator-description
-  sprite:
+  sprite: 
     sprite: Objects/Tools/spy_device.rsi
     state: icon
   content:
   - EncryptionKeyStationMaster
   - CyberPen
+  - SpyCrewMonitor
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/Catalog/thief_toolbox_sets.yml:92]: This was also removed in Resources/migration.yml and needs to be removed here or returned

**Pull request merged**